gym_electric_motor/visualization/motor_dashboard_plots/reward_plot.py

```diff
@@ -30,6 +30,10 @@ def set_env(self, env):
         self._y_lim = (min_limit - spacing, max_limit + spacing)
         self._label = 'reward'
 
+    def reset_data(self):
+        super().reset_data()
+        self._reward_data = np.zeros_like(self._x_data, dtype=float) * np.nan
```

Comment:
Is this producing zeros or nans?

Reply:
This is producing nans. Do you know a more elegant way to initialize a nan-array? Nans are used because they generate a non visible line when passed to matplotlib. When the env simulates, the nans are overwritten with the reward signal.

Reply:
Checkout np.full

https://numpy.org/doc/stable/reference/generated/numpy.full.html

Reply:
Fixed 😃